practices/graded-browser-support.md

```diff
@@ -87,7 +87,7 @@ This means that even if we consider IE10 a Grade-C browser, we currently serve I
 
 [CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) has been available in all evergreen browsers [for the last year](https://caniuse.com/#feat=css-grid). It offers a much-simplified way of working with layout and offers new design opportunities.
 
-However, CSS Grid is not available in the full standardised form in Internet Explorer 10 or 11, both of which are currently *de facto* supported by us (see above).
+However, CSS Grid is not available in the full standardised form in Internet Explorer 10 or 11, both of which are currently *de facto* supported as Grade A browseres by us (see above).
```

Comment:
`browseres` i didnt know you were from cornwall

Reply:
![](https://media.giphy.com/media/E4vIfoWCbdd3a/giphy.gif)

Reply:
I'm getting a speelchicker for my editor.